build/VStudio/dotnet/winfsp.net.csproj

```diff
@@ -1,44 +1,17 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
There seem to be a lot of changes in this file. This may be necessary because of major format changes, but my preference would be for changes to be minimized if possible.

Reply:
The Sdk format almost removes the need for all props by having good defaults. the only property you need is TargetFramework/TargetFrameworks other changes is to override the defaults.

I can see more lines here that can be removed ProjectName can be removed if it is same as the csproj filename.
FileAlignment isnt needed.

To summarize, with Sdk format you can remove almost all props, but you can also keep many of them if the commit diff is more important, But to take full advantage of the format, it is best to try and remove as much as possible.

/Outside reader that loves the project

Reply:
Understood. Perhaps a good compromise is to minimize changes in this PR so that it is easily understood and then remove those redundant properties in future commits.